frontend/resourceadm/app/App.tsx

```diff
@@ -13,7 +13,9 @@ import { ErrorPage } from '../pages/ErrorPage';
 import { RedirectPage } from '../pages/RedirectPage';
 import { SearchField } from '@altinn/altinn-design-system';
 import { ScreenReaderSpan } from 'resourceadm/components/ScreenReaderSpan';
+import { ListAdminPage } from 'resourceadm/pages/ListAdminPage/ListAdminPage';
```

Comment:
We should use absolute path inside the same package for imports 😄 
I know this is not the case many places in resourceadm code, and I don't think we should change them all now, but the newly added imports should be changed to that.

Reply:
changed imports in new files to relative import